homeassistant/components/plex/config_flow.py

```diff
@@ -0,0 +1,215 @@
+"""Config flow for Plex."""
+import logging
+
+import plexapi.exceptions
+import requests.exceptions
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_URL,
+    CONF_HOST,
+    CONF_PORT,
+    CONF_TOKEN,
+    CONF_SSL,
+    CONF_VERIFY_SSL,
+)
+
+from .const import (
+    CONF_SERVER,
+    CONF_SERVER_IDENTIFIER,
+    DEFAULT_PORT,
+    DEFAULT_SSL,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+    PLEX_SERVER_CONFIG,
+    SERVERS,
+)
+from .errors import NoServersFound, ServerNotSpecified
+from .server import PlexServer
+
+_LOGGER = logging.getLogger(__package__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class PlexFlowHandler(config_entries.ConfigFlow):
```

Comment:
Remove the decorator and pass the domain as keyword argument to the class.

https://developers.home-assistant.io/docs/en/config_entries_config_flow_handler.html#defining-your-config-flow

Reply:
This was causing lint issues in my original config flow PR. Will use the new syntax.